tensornetwork/backends/pytorch/pytorch_backend_test.py

```diff
@@ -497,6 +497,12 @@ def test_eigs_not_implemented():
     backend.eigs(np.ones((2, 2)))
 
 
+def test_gmres_not_implemented():
```

Comment:
thanks for writing those tests as well!

Reply:
No problem. I couldn't decide whether the symmetric and shell backend (what is the shell backend?) needed these too, since they do not seem to inherit from AbstractBackend and do not seem to consistently check for NotImplemented.

Reply:
I think they both inherit from `AbstractBackend`, if not then that's a bug

Reply:
Ok, I'll add it then